tendrl/node_agent/tests/test_ansible_runner.py

```diff
@@ -6,12 +6,7 @@
 from ansible import modules
```

Comment:
I think we can delete this file as we no longer have this source in tendrl-node-agent